logitechpjt/urls.py

```diff
@@ -1,15 +1,12 @@
-from django.contrib import admin
 from django.urls import (
     path,
     include
 )
-
 from account import urls
-from product import urls
-from cart import urls
 
 urlpatterns = [
-	path('account', include('account.urls')),
-    path('product', include('product.urls')),
+    path('account/', include('account.urls')),
+    path('product/', include('product.urls')),
+    path('index/', include('product.urls')),
```

Comment:
```suggestion
    path('account', include('account.urls')),
    path('product', include('product.urls')),
    path('index', include('product.urls')),
```
제일 상위 url은 앞뒤로 슬래시 붙이지 않는 것이 wecode coding convention입니다.